contracts/liquidity-pool/src/contract.rs

```diff
@@ -1734,6 +1736,147 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_collateral_check() {
+        // NOTE: available liquidity stays fixed as the test environment does not get changes in
+        // contract balances on subsequent calls. They would change from call to call in practice
+        let available_liquidity_1 = 1000000000u128;
+        let available_liquidity_2 = 2000000000u128;
+        let available_liquidity_3 = 3000000000u128;
+        let mut deps = th_setup(&[
+            coin(available_liquidity_1, "depositedcoin1"),
+            coin(available_liquidity_2, "depositedcoin2"),
+            coin(available_liquidity_3, "depositedcoin3"),
+        ]);
+
+        let exchange_rate_1 = Decimal::from_ratio(13 as u128, 2 as u128);
```

Comment:
```suggestion
        let exchange_rate_1 = Decimal::from_ratio(13u128, 2u128);
```
Just for the sake of consistency, doesn't matter really.